code/modules/mob/living/carbon/life.dm

```diff
@@ -7,11 +7,24 @@
 	if(!loc)
 		return
 
+	if(damageoverlaytemp)
+		damageoverlaytemp = 0
+		update_damage_hud()

```

Comment:
this is to handle damageoverlaytemp stuff, the bigger damage hud overlay appears when update_damage_hud() is first called, then the next time life() is called the damage hud overlay goes back to its normal level.